src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -45,11 +46,32 @@ public static class SimplificationOptions
         public static Option<bool> AllowSimplificationToBaseType { get; } = new Option<bool>(NonPerLanguageFeatureName, "AllowSimplificationToBaseType", true);
 
         /// <summary>
-        /// This option says if we should simplify away the this. or Me. in member access expression
+        /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in member access expressions.
         /// </summary>
+        [Obsolete]
         public static PerLanguageOption<bool> QualifyMemberAccessWithThisOrMe { get; } = new PerLanguageOption<bool>(PerLanguageFeatureName, "QualifyMemberAccessWithThisOrMe", defaultValue: false);
```

Comment:
Well, you've already made the option no longer work. That's also an API breaking change. Perhaps we should keep this this as some sort of parent option that sets or unsets all of the more-specific options underneath it? In the UI, there'd be this checkbox with the various member checkboxes nested underneath.